installer/manifests/keptn/templates/rbac.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
Please also make sure that the corresponding lighthouse-svc service account and rolebinding is not created when the service is disabled.